### PR TITLE
[release-1.0.0-RC3] Docker compose loadbalanced stack

### PR DESCRIPTION
**Pull request by ballinette**
see #58:

> New docker-compose files layout
> 
> * /docker-compose.yml standard Kuzzle stack + Backoffice (static build) without tests nor debug ports.
> * /docker-compose/test.yml standard Kuzzle stack + Backoffice (static build) without debug ports (tests start automatically) [meant for Travis]
> * /docker-compose/dev.yml debug-mode Kuzzle stack + Backoffice (development server) - tests can be run via npm test
>

### CODE REVIEW

**Comment by benoitvidis** [comment]
+1

**Comment by scottinet** [comment]
Why the proxy part of the docker stack is named "loadbalancer"?

**Comment by ballinette** [comment]
agree with @scottinet , we should rename "loadbalancer" by "proxy".
+1 for me as soon as this change is done.

File: docker-compose/dev.yml [hunk 1]
```diff
@@ -1,41 +1,50 @@
...
   ports:
     - "3000:3000"
     - "8082:8080"
     - "9876:9876"
+    - "35729:35729"
   links:
-    - kuzzle
-    # add link to elasticsearch in order to perform a dump
-    - elasticsearch
+    - loadbalancer:kuzzle
```
Comment (dbengsch) [inline docker-compose/dev.yml]: an idea was to rename "kuzzle" in "api", to avoid such misleading container bindings

Comment (benoitvidis) [inline docker-compose/dev.yml]: +1 for me on changing `loadbalancer` to `proxy` only.
If/once done, the bo link `proxy:kuzzle` is fine with me as it is actually quite meaningful.

**Comment by ballinette** [comment]
All requested changes in docker-compose files have actually been merged by PR #60 
Still in this PR: removing of the debug port in docker-compose/dev.yml
I'm not sure it is what we want ? what do you think ?

**Comment by xbill82** [comment]
@ballinette I think this has been done since `node-inspector` doesn't work on the `kuzzleio/kuzzle:develop` image (which extends `kuzzleio/base`, not intended to ship with the inspector).

@benoitvidis and I have found a hack-ish workaround to install `node-inspector` on the `kuzzle` container, so that the debug ports can be exposed. Coming up with next commit...

**Comment by ballinette** [comment]
ok for me as soon as @benoitvidis validates the added command

**Comment by ballinette** [comment]
Oh, another question: why do we use "kuzzleio/kuzzle:develop" image and not "kuzzleio/dev:alpine" for kuzzle server ?

**Comment by xbill82** [comment]
@ballinette because `kuzzleio/dev:alpine` is based on the `kuzzle-containers` repo and assumes you mount the kuzzle repo as a volume.

**Comment by ballinette** [comment]
OK

**Comment by jenow** [comment]
+1, not merging until @benoitvidis answers

**Pull request merged**